src/ralph/accounts/views.py

```diff
@@ -81,12 +85,23 @@ def post(self, request, *args, **kwargs):
         )
         asset.save()
 
-        return super().get(request, *args, **kwargs)
+        response = HttpResponseRedirect(reverse('current_user_info'))
+        response.set_cookie('tag_success', 1)
+        return response
 
 
 class CurrentUserInfoView(UserInfoMixin, RalphBaseTemplateView):
     template_name = 'ralphuser/my_equipment.html'
 
+    def dispatch(self, request, *args, **kwargs):
+        try:
+            if request.COOKIES.get('tag_success'):
+                messages.success(request, _('Asset successfully tagged'))
```

Comment:
move this to ``InventoryTagView.post`` after ``asset.save`` -- django's message framework handle of all stuff to display message to user in next request